Client/Clique/Common/Constants/Constants.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Constants.swift
+//  Clique
+//
+//  Created by Infinum on 19.11.2022..
+//
+
+import UIKit
+
+
+enum Constants{
```

Comment:
Fali razmak između `Constants` i `{`.

```suggestion
enum Constants {
```